StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  PaymentSheetVerticalViewController.swift
+//  StripePaymentSheet
+//
+//  Created by Yuki Tokuhiro on 5/3/24.
+//
+
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewController {
+    var selectedPaymentOption: PaymentSheet.PaymentOption?
+    /// The type of the payment method that's currently selected in the UI, or unknown if no payment method is selected.
+    var selectedPaymentMethodType: PaymentSheet.PaymentMethodType = .stripe(.unknown)
+
+    var delegate: (any FlowControllerViewControllerDelegate)?
```

Comment:
Shouldn't this be `weak`? Also I think this can just be `weak var delegate: FlowControllerViewControllerDelegate?`

Reply:
Oh oops ty, this was auto-generated.